README.md

```diff
@@ -1,7 +1,18 @@
 # Source Repository for graphql.org
 
-This repository contains the source code of https://graphql.org website.
+This repository contains the source code of [GrapQL](https://graphql.org)
```

Comment:
Small typo:

```suggestion
This repository contains the source code of [GraphQL](https://graphql.org)
```

Reply:
I'd also add that this is the source code for the GraphQL _website_ (as to not be confused with the spec)

```suggestion
This repository contains the source code for the [GraphQL website](https://graphql.org).
```